data/hai missions.txt

```diff
@@ -1053,3 +1053,250 @@ mission "Hai Honeymoon"
+			`	Xavier stands up, with a shocked expression. "How? We had a Leviathan escorting them!"`
+			`	After catching his breath, the man at the door responds. "The pirates must have been stealing Hai tech from ships going back home, the freighter's captain said they shredded the escort!" Chaos spreads amongst the other captains, and the group sitting with you is in anguish.`
+			choice
+				`	"Are those art pieces really that valuable?"`
+				`	"Can't you just postpone the festival until the pirates are taken care of?"`
+			`	Xavier collapses back to the table, holding his head in both hands. "Those paintings and sculptures were our main attraction! They're worth dozens of millions! It'll take a month before the Navy ever notices the pirates there and make it safe for the convoy to come here. The art exhibition is scheduled two weeks from now!"`
+			`	As they try to figure out what to do, Nanachi speaks up. "I'll go there, I pilot a human ship and if we just need to transfer cargo from one ship to the other, I could get out of there quickly, nobody would notice that a Hai ever went there."`
+			`	"Are you insane? If they managed to take down that Leviathan, you wouldn't last a second! Besides, our convoy was extremely fast, if even they were caught your chances of outrunning the pirates are next to nothing!" Vanessa says.`
+			`"Captain <last>," Nanachi turns to you. "I know you already helped me way more than anyone would ever be willing to, and I'm really grateful for that, but would you be willing to help me one last time, to get to that planet, and travel back here?" While it's certain that for a pirate fleet to take down a Leviathan so easily they must be incredibly powerful, and this could be extremely dangerous, you're sure that if Nanachi tries to get to Prime by herself she'll just get herself killed.`
+			choice
+				`	"Alright, I'll make sure we reach Prime and get back here safely."`
+				`	"No, I agreed to help you find and deal with some simple jobs, I can't risk fighting such a fleet by myself."`
+					decline
+			`	She thanks you profusely, and promises that she'll pay you for everything as soon as she can. The organizers are a bit hesitant, but seeing as no other captain here decided to offer them help, they agree to let you two go to Prime retrieve the cargo.`
+			`	As you're leaving the bar, Xavier pulls you aside, speaking very quietly, "<last>, I know you're just trying to help this girl, but don't get killed over this, if you run into those pirates don't try to fight them, just rush to Prime."`
+				accept
+	npc
+		personality nemesis waiting
+		government "Pirate"
+		system "Betelgeuse"
+		ship "Marauder Firebird (Hai)" "Nut King Call"
+		ship "Marauder Splinter (Hai)" "The Nutcracker"
+		ship "Marauder Quicksilver (Hai)" "Hazelnut's Bane"
+		ship "Marauder Bounder (Hai)" "Rodent Hunter"
+	npc accompany save
```

Comment:
I get that it's to make the mission more challenging, but it makes no sense at all that Nanachi would follow you in her ship when it has no combat power and would only make the situation vastly more dangerous for her.